weblate/accounts/tests/test_views.py

```diff
@@ -551,6 +552,24 @@ def test_unsubscribe(self):
         subscription.refresh_from_db()
         self.assertEqual(subscription.frequency, FREQ_NONE)
 
+    def test_profile_password_warning(self):
+        with mock.patch.object(User, "has_usable_password", return_value=False):
+            response = self.client.get(reverse("profile"))
+            self.assertContains(response, "Please enable the password authentication")
+            with modify_settings(
+                AUTHENTICATION_BACKENDS={
+                    "remove": "social_core.backends.email.EmailAuth"
+                }
+            ):
+                load_backends(settings.AUTHENTICATION_BACKENDS, force_load=True)
+                response = self.client.get(reverse("profile"))
+                self.assertNotContains(
+                    response, "Please enable the password authentication"
+                )
+        self.assertEqual(self.user.has_usable_password(), True)
+        response = self.client.get(reverse("profile"))
+        self.assertNotContains(response, "Please enable the password authentication")
```

Comment:
Due to caching in python-social-auth, the backends need to be loaded again to avoid breaking rest of the tests:

```suggestion
        self.assertNotContains(response, "Please enable the password authentication")
        load_backends(settings.AUTHENTICATION_BACKENDS, force_load=True)
```